### ISSUE TITLE
Add-On for Firefox?

### ISSUE THREAD

**Issue by webdevler** [AdguardTeam/StealthMode#17]
Any chance you're working on a version of the Chrome extension for Firefox?  Would love to use this on there as well.

**Comment by ameshkov** [comment]
Yeah, since FF now supports WebExtensions, it should be relatively easy to implement.

The problem is that Stealth Mode add-on is quite outdated comparing with it's Windows module counterpart, we'd better improve it also. Hope we'll get some time to finally do it soon.

**Comment by ultravio1et** [comment]
be great if you could.

plus also maybe update adguard so it can be used in multi-process firefox
